lib/client-assets.php

```diff
@@ -317,6 +317,52 @@ function gutenberg_extend_wp_api_backbone_client() {
 			} ) );
 		};
 JS;
+
+	/**
+	 * @var WP_REST_Server $wp_rest_server
+	 */
+	global $wp_rest_server;
+
+	// Ensure the rest server is initialized.
+	if ( empty( $wp_rest_server ) ) {
```

Comment:
Should this block not be replaced with `$wp_rest_server = rest_get_server()`, and then eliminate the `global`?